app/controllers/source_sets_controller.rb

```diff
@@ -67,4 +67,11 @@ def source_set_params
                                        author_ids: [],
                                        tag_ids: [])
   end
+
+  ##
+  # @return [Array<Tag>]
+  def get_tags_from_params
+    return nil unless params[:tags].present?
+    params[:tags].map { |slug| Tag.find_by_slug(slug) }.compact
```

Comment:
Is there a way to have the query work on a list of values, instead of running individually for each value?

For example, is this sane?: `Tag.where("slug IN (?)", params[:tags])`

It's probably not a big deal with a small list of tags, but I'm trying to think of ways to reduce the number of database queries.